src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -2657,6 +2665,34 @@ public boolean isEphemeral()
             return ephemeral;
         }
 
+        /**
+         * Gets the waveform data encoded in this attachment. This is currently only present on
+         * {@link MessageFlag#IS_VOICE_MESSAGE voice messages}.
+         *
+         * @return A possibly-{@code null} array of integers representing the amplitude of the
+         *         audio over time. Amplitude is sampled at a rate of 10Hz. The values in this
```

Comment:
Apparently, this sampling rate changes on longer voice messages. So it probably has to be determined using `getDuration` and the length of the array.